instantsearch.voice/src/main/java/com/algolia/instantsearch/voice/ui/VoiceDialogFragment.kt

```diff
@@ -16,12 +16,14 @@ import kotlinx.android.synthetic.main.layout_voice_overlay.view.*
 
 @SuppressLint("InflateParams")
 class VoiceDialogFragment : DialogFragment(), VoiceInput.VoiceInputPresenter {
-    val input: VoiceInput by lazy { VoiceInput(this) }
+
     private val content: View by lazy {
         LayoutInflater.from(activity).inflate(R.layout.layout_voice_overlay, null)
     }
-
     private var suggestions: List<String> = emptyList()
+
+    val input: VoiceInput by lazy { VoiceInput(this) }
```

Comment:
I get why adding a newline after the class declaration, but why moving this between `suggestions` and `setSuggestions?` :thinking:

Reply:
my OCD striked again. I try to sort variables by visibility and type (private with private, var with var ..). You can pretty much ignore this.